### DISCUSSION TITLE
What if we hit the Org Limits ?

### DISCUSSION THREAD

**Discussion by Doondireddy** [jongpie/NebulaLogger#294] [Q&A]
What if we hit the Org Limits ? 
Turning off using custom settings is only the option OR do you have any other suggestions?

**Comment by jongpie** [comment]
Hi @Doondireddy - could you provide some details about what limits you're hitting? Depending on what specific limit(s) you're encountering, there may be a few ways you can mitigate the issues.

**Reply by ksamanth** [reply]
Hi @jongpie , This could be unrelated to this thread but is along the same lines. I was trying to handle CPU time limit exception. I know we need to code better to handle it, but still if in some case I end up hitting this exception / any limit exception for that matter, how to handle / log it. is it possible at all?

**Reply by jongpie** [reply]
Hi @ksamanth - unfortunately, the limit exceptions (including CPU time limit) cannot be caught in Apex. If you are using the `EVENT_BUS` save method in Nebula Logger (the default save method), then any log entries that you've saved before hitting the CPU time limit should still be saved, but there's no way to catch the actual CPU time limit exception.
